README.md

```diff
@@ -7,22 +7,35 @@ Installation
 ------------
 
 Just add this line to your `Gemfile`:
+
 ```ruby
 gem "solidus_auth_devise"
 ```
 
-Then run `bundle install`.
+Then, run `bundle install`.
+
+After that's done, you can install and run the necessary migrations, then seed the database:
+
+```shell
+bundle exec rake railties:install:migrations
```

Comment:
This would also install migration of all other engines mounted in the app, where `bin/rake solidus_auth:install:migration` only installs this engines migration files. 

I'm aware that the former version is promoted everywhere, but it could lead to unexpected behaviour. When we change this now, we should correct this right now.

Also note the usage of `bin/rake` as this is the preferable Rails way to run binstubs in Rails 4+ apps. This takes care of bundler and/or spring usage.